Sources/DesignKit/Views/WebView.swift

```diff
@@ -6,18 +6,22 @@ import SwiftUI
 import WebKit
 
 // TODO: at this moment, this just loads an url in a web view. We will add a proper recommendation view after beta
-struct RecommendationDetailView: UIViewRepresentable {
-    let recommendation: Recommendation
+public struct WebView: UIViewRepresentable {
+    public let url: String
 
-    func makeUIView(context: Context) -> WKWebView {
+    public init(url: String) {
+        self.url = url
+    }
+
+    public func makeUIView(context: Context) -> WKWebView {
         let webview = WKWebView()
         webview.backgroundColor = UIColor(.mosoLayerColor1)
         return webview
     }
 
-    func updateUIView(_ webView: WKWebView, context: Context) {
+    public func updateUIView(_ webView: WKWebView, context: Context) {
         // TODO: this is just an example fallback, we need to add a proper error view
-        let url = URL(string: recommendation.url) ?? URL(string: "https.getpocket.com")!
+        let url = URL(string: url) ?? URL(string: "https.getpocket.com")!
```

Comment:
Maybe we should load a "blank" page instead of defaulting to the Pocket page. Ideally, there'd be something else as you mentioned, but I hesitate to have the "error" state be the Pocket homepage.

Reply:
I just migrated Gio's code.
Maybe we move the URL init out of the WebView, so if that fails the previous view can handle it?

Reply:
The initializer of the SwiftUI view should be similar to that of the underlying WKWebView, so I agree that it should take in a URL. I'm still wondering, though, if it's worth having the reading list kit be the first of the currently two spots that could use SFSafariViewController instead of a web view.